test.sh

```diff
@@ -51,6 +62,9 @@ echo "${PREFIX} Platform: ${PLATFORM}"
 
 printf "${PREFIX} Building container\n"
 
+printf "${PREFIX} using goss (${GOSS_PATH})\n"
+printf "${PREFIX} using dgoss (${DGOSS_PATH})\n"
```

Comment:
@bryanlatten : Is there a reason we are using `printf` here? since in lines 57-59 we're using echo (which already has the newline)? Maybe for consistency we should stick to just `echo` unless there's a specific reason we need a different way?

Reply:
it was only consistency, we could switch one or switch them all. your call

Reply:
Thanks @bryanlatten . Yes it would be worthwhile to have them be consistent but we can deal with it some other time. Like I mentioned not enough to block the PR.